organizations/models.go

```diff
@@ -10,6 +10,7 @@ import (
 const (
 	OrganizationCollectionName     = "organizations"
 	InstalledPluginsCollectionName = "installed_plugins"
+	OrganizationSettings = "organizations_settings"
```

Comment:
I was thinking we let the setting sit inside the organization collection, it wouldn't be too large of a data that we will need a separate collection for all settings.